deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -146,5 +146,13 @@ def collect_screenshots_for_languages(path, ignore_validation)
 
       return screenshots
     end
+
+    def self.available_languages
```

Comment:
Interesting. Why was this change necessary?

Reply:
I copied the same approach from loader.rb 

https://github.com/liamnichols/fastlane/blob/2fabb8ec9f84a19ebb00af47ac6875ca4fdb576f/deliver/lib/deliver/loader.rb#L27-L31

Since this code path will run through the tests, I didn’t want it to try and fetch this via the ITC api...

Although now you point it out, I guess I could just mock that spaceship method in the spec instead? Would it be better to do that? 🤔

Reply:
It can be `private`

Reply:
I think I prefer the idea of mocking it now that you mention it and since that is what I’m doing for the file size it probably makes even more sense... I’ll update the branch to do this before merging 👍🏼

Reply:
So for some reason `Spaceship::Tunes.client` is `nil` during the tests so it involves more than just stubbing the `available_languages` property on the class as I have to also initialise it in the tests and I'm not really sure how to do that 😅 

I've just reverted it back to how it was and used the `Helper.test?` check again

Reply:
That's fine, maybe create an issue for what you tried to achieve so we don't forget about it and can maybe take another shot later?